roboRIO-2024/src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -96,7 +100,7 @@ public Drivetrain(Settings settings) {
     rightLeader.setSmartCurrentLimit(60);
     leftLeader.setSmartCurrentLimit(60);
     rightLeader.setSmartCurrentLimit(60);
-    applytoAllMotors((motor) -> {motor.setOpenLoopRampRate(0.2);});
+    // applytoAllMotors((motor) -> {motor.setOpenLoopRampRate(0.2);});
```

Comment:
I realized you should do 
```

applytoAllMotors((motor) -> {motor.setOpenLoopRampRate(0);});

```

because the previous setting is saved in flash on the SparkMax.